react/week3/my_first_app/src/components/Counter.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+class Counter extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        value: this.props.initialValue
+      };
+    }
```

Comment:
In the version of React we're using this could be rewritten as 
```suggestion
    state = {
        value: this.props.initialValue
      };
```

It's not clear from the docs but react takes care of the constructor behind the scenes!